proto/vendorinfo/tx.proto

```diff
@@ -21,7 +21,7 @@ message MsgCreateVendorInfo {
 
   string vendorName = 3 [(gogoproto.moretags) = "validate:\"required,max=128\""];
   string companyLegalName = 4 [(gogoproto.moretags) = "validate:\"required,max=256\""];
-  string companyPrefferedName = 5 [(gogoproto.moretags) = "validate:\"omitempty,max=256\""];
+  string companyPrefferredName = 5 [(gogoproto.moretags) = "validate:\"omitempty,max=256\""];
```

Comment:
The field name should be `companyPreferredName` not `companyPrefferredName`

Reply:
Done! ✅